prow/e2e-suite.sh

```diff
@@ -58,10 +58,6 @@ ISTIO_GO=$(cd $(dirname $0)/..; pwd)
 export HUB=${HUB:-"gcr.io/istio-testing"}
 export TAG="${GIT_SHA}"
 
-if [[ -z "${E2E_ARGS:-}" ]]; then
-  E2E_ARGS="$@"
-else
-  E2E_ARGS+=("$@")
-fi
+E2E_ARGS+=" $@"
```

Comment:
there will be bugs when spaces are in the arguments but I guess otherwise/in meantime this should work
(should really be ${@})